README.md

```diff
@@ -81,19 +81,19 @@ You can write write a matrix to EXR without specifying channel names. Default ch
 | 3 channels  | R, G, B    |
 | 4 channels  | R, G, B, A |
 
+The precision is `FLOAT` as default if the argument `precision` is not set, even though the numpy type is `float16`.
```

Comment:
True ... I don't think this is great behavior, though. What do you think? Should it be changed?

Reply:
Acturally I wanted to change this. But considering `read()` and `InputFile.get()` ignore the data type in EXR file either, I though `write()`'s behavior is OK. 

Maybe input and output can be both changed or both left unchanged.

Reply:
I think it would be best if we try to match the input type. When I wrote this code, I only really cared about 32bit floats, but I think that for people new to this library the behavior you propose makes more sense.

So, we try to find a matching float or integer type, and otherwise try to cast to float?

Do you want to update this pull request, or shall I do it?

Reply:
Let me do it. :smile: